docker/packager/packager

```diff
@@ -168,6 +168,7 @@ def parse_env_variables(
             "-DCMAKE_TOOLCHAIN_FILE=/build/cmake/darwin/toolchain-x86_64.cmake"
         )
         result.append("EXTRACT_TOOLCHAIN_DARWIN=1")
+        result.append("EXPORT_SOURCES_WITH_SUBMODULES=1")
```

Comment:
so, it will be always in the `clickhouse-builds`, but only for darwin build. Looks reasonable

Reply:
Only for one of 2 darwin builds. It was pretty fast in CI.

Reply:
Just found, that we don't have the same thing for `is_cross_darwin_arm`

Reply:
Got it, the artifact will be the same, so we need only one of them